src/copy.js

```diff
@@ -7,8 +7,18 @@ chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
   }
 });
 
+const FIX_TARGET = /twitter\.com|x\.com/g
+
 document.addEventListener('copy', e => {
-  currentLink = document.location.href;
+  currentLink = window.getSelection().toString();
+
+  if (!currentLink) {
+    return;
+  }
+
+  if (!currentLink.match(FIX_TARGET)) {
+    return;
+  }
```

Comment:
If the copied content is not a twitter link, it will be returned early.
This will also resolve the content of this comment.
https://github.com/thevenuz/xify/issues/9#issuecomment-2277743849